tools/benchmark/module/depinject.go

```diff
@@ -0,0 +1,69 @@
+package module
+
+import (
+	"unsafe"
+
+	"cosmossdk.io/depinject"
+
+	modulev1 "cosmossdk.io/api/cosmos/benchmark/module/v1"
+	"cosmossdk.io/core/appmodule"
+	"cosmossdk.io/core/store"
+	"cosmossdk.io/depinject/appconfig"
+	"cosmossdk.io/log"
+	gen "cosmossdk.io/tools/benchmark/generator"
+)
+
+const ModuleName = "benchmark"
+const maxStoreKeyGenIterations = 100
+
+func init() {
+	// TODO try depinject gogo API
```

Comment:
Could you try this TODO now? The depinject gogo api works: https://github.com/cosmos/cosmos-sdk/blob/v0.52.0-beta.2/testutil/x/counter/depinject.go#L21-L27

Reply:
I'm going go try this in a follow up PR!